lib/Table/TableStyle.ts

```diff
@@ -145,6 +149,58 @@ export default class TableStyle {
     return this.styleTraits.time;
   }
 
+  /** Compute rectangle for point (lat/long) based table styles */
+  @computed get rectangle() {
+    if (this.isPoints()) {
+      const bounds: BBox = [Infinity, Infinity, -Infinity, -Infinity];
+
+      if (this.longitudeColumn && this.latitudeColumn) {
+        for (let i = 0; i < this.longitudeColumn.values.length; i++) {
+          const long = this.longitudeColumn.valuesAsNumbers.values[i];
```

Comment:
I would probably use `reduce` here and do this functionally but hey, a for loop works. 🤷

Reply:
I always prefer for loops in table mixin/style code for performance